public/src/components/channelManagement/epicTests/utils/defaults.ts

```diff
@@ -21,7 +21,6 @@ export const DEFAULT_SECONDARY_CTA: Cta = {
 
 const DEV_AND_CODE_DEFAULT_VARIANT: EpicVariant = {
   name: 'CONTROL',
-  heading: "Since you're here ...",
```

Comment:
this is pre-populated in CODE/DEV, but it's best to keep it closer to how the tool is used in PROD